test/logger_test.rb

```diff
@@ -455,9 +455,9 @@ class TestLogger < Hanami::Logger; end
           output =
             stub_stdout_constant do
               class TestLogger < Hanami::Logger; end
-              TestLogger.new.info('This is a message with spaces')
+              TestLogger.new.info("This is a message with quotes \" ' and with spaces")
```

Comment:
@jodosha with that the test passes, of course we can make a gsub to scape inner single quotes if we wrap with them or viceversa.